### PR TITLE
Spellchecking and removing i parameter

### PR DESCRIPTION
**Pull request by deknos**
saw some spelling errors :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -508,7 +508,7 @@ $
 To wipe everything clean:
 
 ```bash
-rm -rfi ~/.binenv ~/.config/binenv/
```
Comment (leucos) [inline README.md:511]: This is intentional so the use has to confirm removal explicitely. Better be on the safe side :D

**Review by leucos — CHANGES_REQUESTED** [review]
Thanks, this is awesome. I would like to keep the `-rfi` though.

File: README.md (hunk 1 above)
Comment (deknos) [reply README.md:511]: but why then the -f? that's kind of contradictory, but okay, i will adapt it :D

**Comment by leucos** [comment]
Great thanks ! Note that you might have to rebase your other PR

File: README.md (hunk 1 above)
Comment (leucos) [reply README.md]: > but why then the -f? that's kind of contradictory, but okay, i will adapt it :D

Agreed that in the context, it does not make sense. I will remove it.

**Pull request merged**